tests/yam/agama/boot_agama.pm

```diff
@@ -58,6 +59,17 @@ sub prepare_boot_params {
     return @params;
 }
 
+sub validate_rescue_system {
+    assert_screen('inst-console', 30);
```

Comment:
this shouldn't be needed.
Thanks for showing the possibility to do it here, I just realize that headles is not the same that this task, this is more like check medium where you have a different flow and a different schedule, it is not a installation, therefore we can have the validation in a separate module as you wanted :) Take into account we should also check the masked service, so the validation might be a bit longer still.